pkg/app/src/components/Account/modules/identitySection.tsx

```diff
@@ -13,16 +13,17 @@ import {
 	shortAccountAddress,
 } from 'src/utils/accountUtils'
 import { createInfoNotification } from 'src/utils/notificationUtils'
+import { useRouter } from 'next/router'
 
 interface IdentitySectionProps {
 	accountState: AccountState
-	setCurrentTab: (AccountTabs) => void
 }
 
-const IdentitySection: FC<IdentitySectionProps> = ({ accountState, setCurrentTab }) => {
```

Comment:
1. Do not write FC components.
2. Do not pass AccountState or Values you get from a hook as parameter (avoid useless state management)